src/main/java/br/com/edg/project/dao/EstoqueDAO.java

```diff
@@ -0,0 +1,175 @@
+     */
+    public static boolean excluir(Produto prod) {
+       
+        String query = "DELETE FROM ESTOQUE WHERE ID_PRODUTO = ?";
+        
+        PreparedStatement stmt = null;
+        
+        try {
+            Class.forName(Driver);
+            conexao = DriverManager.getConnection(url, user, senha);
+            stmt = conexao.prepareStatement(query);
+            
+            stmt.setInt(1, prod.getCodProduto());
+            
+            int linhasAfetadas = stmt.executeUpdate();
+            
+            if(linhasAfetadas > 0) {
+                return true;
+            } else {
+                return false;
+            }
+        } catch (SQLException ex) {
+            Logger.getLogger(EstoqueDAO.class.getName()).log(Level.SEVERE, null, ex);
+        } catch (ClassNotFoundException ex) {
+            Logger.getLogger(EstoqueDAO.class.getName()).log(Level.SEVERE, null, ex);
+        } finally {
+            try {
+                if(stmt != null)
+                    stmt.close();
+                if(conexao != null)
+                    conexao.close();
+            } catch (SQLException ex) {
+            }
+        }
+        return false;
+    }
+    
+    /**
+     * Método para alterar um produto, recebe objeto Id como referência e os demais objetos para alteração
+     * @param prod - objeto para referenciar produtos que serão alterados
+     * @return - true para alterado e false para não alterado
+     * @throws ClassNotFoundException - Não achou o driver
+     * @throws SQLException - Erro ao tentar conectar-se à base de dados
+     */
+    public static boolean alterar(Produto prod){
+        
+        String query = "UPDATE PRODUTO SET NOME_PRODUTO = ?, VALOR = ?, KG = ?, QUANTIDADE = ? WHERE ID_PRODUTO = ?";
+        
+        PreparedStatement stmt = null;
+        
+        try {
+            Class.forName(Driver);
+            conexao = DriverManager.getConnection(url, user, senha);
+            stmt = conexao.prepareStatement(query);
+            
+            stmt.setString(1, prod.getNomeProduto());
+            stmt.setDouble(2, prod.getValorProduto());
+            stmt.setDouble(3, prod.getQtdePorKg());
+            stmt.setInt(4, prod.getQtdeProduto());
+            stmt.setInt(5, prod.getCodProduto());
+            
+            int linhasAfetadas = stmt.executeUpdate();
+            
+            if(linhasAfetadas > 0) {
+                return true;
+            } else {
+                return false;
+            }
+        } catch (SQLException ex) {
+                Logger.getLogger(EstoqueDAO.class.getName()).log(Level.SEVERE, null, ex);
+        } catch (ClassNotFoundException ex) {
+            Logger.getLogger(EstoqueDAO.class.getName()).log(Level.SEVERE, null, ex);
+        } finally {
+            try {
+                if(stmt != null)
+                    stmt.close();
+                if(conexao != null)
+                    conexao.close();
+            } catch (SQLException ex) {
```

Comment:
Utilizar exemplo já utilizado em outro método feito por você throw new SQLException("Código do produto não existe ou banco de dados vazio.");

Reply:
Fiz um novo push com todas alterações que foram pedidas, veja agora se está correto, caso não esteja me dê um retorno. please!!!

thank you for you feedback!